docs/atlascli/command/atlas-streams-instances-delete.txt

```diff
@@ -0,0 +1,110 @@
+.. _atlas-streams-instances-delete:
+
+==============================
+atlas streams instances delete
+==============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Delete an Atlas Stream Processing instance.
+
+The command prompts you to confirm the operation when you run the command without the --force option.
+
+Before deleting an Atlas Streams Processing instance, you must first stop all processes associated with it.
+To use this command, you must authenticate with a user account or an API key that has the Project Owner role.
+
+Syntax
+------
+
+.. code-block::
+   :caption: Command Syntax
+
+   atlas streams instances delete <name> [options]
+
+.. Code end marker, please don't delete this comment
+
+Arguments
+---------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - name
+     - string
+     - true
+     - Name of the Atlas Streams instance.
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - --force
+     - 
+     - false
+     - Flag that indicates whether to skip the confirmation prompt before proceeding with the requested action.
+   * - -h, --help
+     - 
+     - false
+     - help for delete
+   * - --projectId
+     - string
+     - false
+     - Hexadecimal string that identifies the project to use. This option overrides the settings in the configuration file or environment variable.
+
+Inherited Options
+-----------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -P, --profile
+     - string
+     - false
+     - Name of the profile to use from your configuration file. To learn about profiles for the Atlas CLI, see https://dochub.mongodb.org/core/atlas-cli-save-connection-settings.
+
+Output
+------
+
+If the command succeeds, the CLI returns output similar to the following sample. Values in brackets represent your values.
+
+.. code-block::
+
+   Atlas Streams processor instance '<Name>' deleted
+   
+
+Examples
+--------
+
+.. code-block::
+
+   # Remove an Atlas Stream Processing instance after prompting for a confirmation:
+   atlas streams delete myProcessorInstance
+
+   
+.. code-block::
+
+   # Remove an Atlas Stream Processing instance named myProcessorInstance without requiring confirmation:
+   atlas streams delete myProcessorInstance --force
```

Comment:
atlas streams instance

Reply:
Resolved in [82c2186](https://github.com/mongodb/mongodb-atlas-cli/pull/2138/commits/82c2186c49ac36d4c34bfb1057ac3be1bb66941d)